target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -49,6 +49,9 @@ antrouter-r1)
 	ucidef_set_led_wlan "wlan" "WLAN" "$board:green:wlan" "phy0tpt"
 	ucidef_set_led_default "btc" "BTC" "$board:green:btc" "0"
 	;;
+ap531b0)
+        ucidef_set_led_wlan "wlan" "WLAN" "$board:green:wlan" "phy0tpt"
```

Comment:
@heinzek 
thank u for your help and your patience

what`s the file attr  on your system, i`ve checked manytimes .. the original file in my system is 664....

Thanks alot